packages/shared/lib/transactions.ts

```diff
@@ -31,13 +37,13 @@ export type TokenDisplayBalances = Omit<TokenBalances, 'balance' | 'lockedBalanc
   lockedBalance: bigint
 }
 
-export type AssetInfo = TokenInfo & { verified?: boolean }
+export type AssetInfo = FungibleTokenMetadata & { verified?: boolean }
 
 export type Asset = TokenDisplayBalances & Optional<AssetInfo, 'symbol' | 'name'>
 
-export type VerifiedAsset = Required<Asset>
+export type VerifiedAsset = Required<Asset> & { logoUri: string }
 
-export type UnverifiedAsset = Optional<VerifiedAsset, 'logoURI'>
+export type UnverifiedAsset = Required<Asset> & { logoUri: never }
```

Comment:
I wonder if these changes break anything in our other apps.

Reply:
Should be fine, no more error when running `turbo compile`.